plugins/plugin-maven/che-plugin-maven-server/src/test/java/org/eclipse/che/api/project/server/TestWorkspaceHolder.java

```diff
@@ -10,46 +10,38 @@
  *******************************************************************************/
 package org.eclipse.che.api.project.server;
 
-import org.eclipse.che.api.core.ServerException;
-import org.eclipse.che.api.workspace.shared.dto.WorkspaceConfigDto;
-import org.eclipse.che.api.workspace.shared.dto.WorkspaceDto;
-import org.eclipse.che.dto.server.DtoFactory;
-
-import java.util.ArrayList;
-import java.util.Collection;
-
 /**
  *
  * @author Evgen Vidolob
  */
 //I have no idea why addProject, updateProject and removeProjects is package private,
 // but I need to override WorkspaceHolder to use him in tests
-public class TestWorkspaceHolder extends WorkspaceHolder {
-    public TestWorkspaceHolder() throws ServerException {
-        super(DtoFactory.newDto(WorkspaceDto.class).withId("id")
-                        .withConfig(DtoFactory.newDto(WorkspaceConfigDto.class)
-                                              .withName("name")
-                                              .withProjects(new ArrayList<>())));
-    }
-
-    @Override
-    void addProject(RegisteredProject project) throws ServerException {
-        if (!project.isDetected()) {
-            workspace.addProject(project);
-        }
-    }
-
-    @Override
-    public void updateProject(RegisteredProject project) throws ServerException {
-        if (!project.isDetected()) {
-            workspace.updateProject(project);
-        }
-    }
-
-    @Override
-    void removeProjects(Collection<RegisteredProject> projects) throws ServerException {
-        projects.stream()
-                .filter(project -> !project.isDetected())
-                .forEach(workspace::removeProject);
-    }
-}
+//public class TestWorkspaceHolder extends WorkspaceHolder {
```

Comment:
why the test class is commented ? test should work. If it's dead code, then it needs to be removed